packages/@apphosting/adapter-angular/src/interface.ts

```diff
@@ -12,6 +12,19 @@ export interface OutputBundleOptions {
   needsServerGenerated: boolean;
 }
 
+// Environment variable schema for bundle.yaml outputted by angular adapter
+export interface EnvironmentVariable {
+  variable: string;
+  value: string;
+  availability: availability[]; // currently support RUNTIME only
+}
+
+// defines whether the environment variable is buildtime, runtime or both
+export enum availability {
+  BUILDTIME = "BUILDTIME",
+  RUNTIME = "RUNTIME",
```

Comment:
If we only support RUNTIME should we reflect that in availability too? So we'd have RUNTIME be the only option

Reply:
Sounds good, I changed the env var definition to have runtime only env var rn